### ISSUE TITLE
nise-module not up to date in current release

### ISSUE THREAD

**Issue by mbaumgartl** [sinonjs/sinon#1566]
I try to use the `XMLHttpRequest.timeout` feature which was implemented in version 1.1.0 of nise.

Simple demo:
```html
<html>
<head>
     <link type="text/css" rel="stylesheet" href="mocha.css">
</head>
<body>
    <div id="mocha"></div>
    <script src="mocha.js"></script>
    <script src="chai.js"></script>
    <script src="sinon-3.3.0.js"></script>
    <script>mocha.setup('bdd')</script>
    <script>
    describe('xhr', () => {
        let xhr;
        let clock;

        beforeEach(() => {
            xhr = sinon.useFakeXMLHttpRequest();
            clock = sinon.useFakeTimers();
        });

        afterEach(() => {
            xhr.restore();
            clock.restore();
        });

        it('should support timeout', () => {
            chai.expect(new XMLHttpRequest())
                .to.be.instanceOf(sinon.FakeXMLHttpRequest)
                .and.to.have.property('timeout');
        });
    });
    mocha.run();
    </script>
</body>
</html>
```
Seems that nise module was not up to date during last build.

**Comment by fatso83** [comment]
Feel free to make a pull request ... Safest way of getting it shipped.

**Comment by fatso83** [comment]
And you can also use `nise` standalone too, so you can use it today!

**Comment by mbaumgartl** [comment]
I don't know exactly how a build is made. If `build.js` is executed on a developer machine, sinon.js-package will contain dependency versions installed at this time. It's up to the developer to run `npm update` before the build. I don't know how to fix this in a pull request.

**Comment by fatso83** [comment]
Actually fixed in cf32e6dbc518b250c1f4c176fc07ec5c0c7b56ed. Will be out in 4.0.
